coreos-meta-translator/trans.py

```diff
@@ -102,6 +102,16 @@ def artifact(i):
                     "image": ami_dict["hvm"]
                 }
 
+        # GCP specific additions
+        if input_.get("gcp", None) is not None:
+            arch_dict["media"]["gcp"] = arch_dict["media"].get("gcp", {})
+            arch_dict["media"]["gcp"]["image"] = arch_dict["media"]["gcp"].get("image", {})
+            arch_dict["media"]["gcp"]["image"].update(input_.get("gcp", {}))
+            if arch_dict["media"]["gcp"]["image"].get("image", None) is not None:
```

Comment:
Should this be conditional? Feels like it should be an error if `image` wasn't in the input, no? IOW, can we drop this and instead just let `pop` error out if it's missing?

Reply:
Emm.. I think this condition will always be true, since `arch_dict["media"]["gcp"]["image"].update(input_.get("gcp", {}))` made sure it is at least a `{}`. Anyway this is redundant, dropping this

Reply:
Updated